common/models/user.js

```diff
@@ -1,3 +1,33 @@
+var mailerService = require('../../server/server-utils/emailProvider');
+
 module.exports = function(user) {
+    var User = user;
+
+
+    User.on('resetPasswordRequest', function(info){
+      var MailService = new mailerService();
+      var accessToken = info.accessToken.id;
+      var email = info.email;
+      var user_id = info.accessToken.userId;
+
+      var resetURL = 'http://localhost:8080/resetPassword/?id=' + user_id + '&token=' + accessToken;
```

Comment:
Done, replaced with another environment variable. PR post updated to reflect it. At some point it might be nice to automate this by passing the front-end URL to the backend automatically, but this is the simplest solution I know of right now. Let me know if you'd like me to develop it further.